packages/common/src/index.ts

```diff
@@ -50,3 +50,6 @@ export * from "d3-selection";
 export * from "d3-time-format";
 export * from "d3-transition";
 export * from "d3-zoom";
+
+import { event as _d3Event } from "d3-selection";
+export const d3Event = () => _d3Event;
```

Comment:
How does wrapping this in a function affect the usage? (other than needing to call it as a function)

Reply:
There is a recurring issue where there ends up being two instances of "d3.event" in different scopes and only one is valid - this ensures that "there is only one"...